politeiad/testpoliteiad/convert.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package testpoliteiad
+
+import (
+	v1 "github.com/decred/politeia/politeiad/api/v1"
+	"github.com/decred/politeia/politeiad/cache"
+)
+
+func convertRecordStatusToCache(status v1.RecordStatusT) cache.RecordStatusT {
```

Comment:
Can we not reuse the actual functions that exist for this?

Reply:
They don't exist.

politeiad converts from: 
politeiad v1 > backend > cache

Since the test server doesn't implement a backend it convert from:
politeiad v1 > cache